Invoices.elm

```diff
@@ -2,10 +2,11 @@ module Invoices exposing (..)
 
 import Html exposing (..)
 import Html.Attributes exposing (..)
-import Html.Events exposing (onInput, onClick)
+import Html.Events exposing (onClick, onInput)
 import Round
 
 
+main : Program Never Model Msg
```

Comment:
just type definition of Program. The interesting thing here is `Never` type which is the type which can't be constructed (you can think of it as about type without constructor). It indicates that Program is something that never halts. [There is some desc about the same thing within typescript](https://basarat.gitbooks.io/typescript/content/docs/types/never.html) if you're interested.